pages/_app.tsx

```diff
@@ -51,6 +51,17 @@ function MyApp<MyAppProps>({ Component, pageProps }) {
   const formMethods = useForm<FormInputData>({ mode: "onBlur" });
   const { favIconPath, appTitle } = appConfig;
 
+  // Setting the "lang" and the "dir" attribute
+  const { i18n } = useTranslation("common");
+  React.useEffect(() => {
+    let lang = query.lang !== undefined ? query.lang : "en";
```

Comment:
Not needed but you can use the typescript shorthand for testing for undefined properties:
```
let lang = query?.lang || "en";
```